app/views/stations/show.erb

```diff
@@ -2,5 +2,12 @@
 
 <h1> <%= @station.name %> </h1>
 <p> <%= @station.city %> </p>
-<p> <%= @station.dock_count %> </p>
-<p> <%= @station.installation_date%> </p>
+<p> Dock Count: <%= @station.dock_count %> </p>
+<p> Installation Date: <%= @station.installation_date%> </p>
+<p> Number of starting rides: <%= Station.number_of_starting_rides(@station.id) %></p>
+<p> Number of ending rides: <%= Station.number_of_ending_rides(@station.id) %></p>
+<p> Most frequent destination: <%= Station.most_frequent_destination_station(@station.id) %>
+<p> Most frequent origination: <%= Station.most_frequent_origination_station(@station.id) %>
```

Comment:
This is also a code smell. Calling methods on the model in the view i.e. `Station.number_of_starting_rides` so push that logic down to the model. These also seem like instance methods that you are calling on the class, could they just be instance methods? Then you dont have to pass an id to it, it already knows about `self`